app/serializers/api/public/v1/serializer_service.rb

```diff
@@ -0,0 +1,14 @@
+module API
+  module Public
+    module V1
+      class SerializerService
+        def execute
+          {
+            Course: API::Public::V1::SerializableCourse,
+            Provider: API::Public::V1::SerializableProvider,
```

Comment:
Just to note, we don't serialize the provider in this PR, so we don't need this. We will need to add `recruitment_cycle` and `locations`, however, when we add support for inclusion.

Optional: Remove `Provider` here and re-add it when we add inclusion.

Reply:
It will get re-added soon, so i do see the benefit of removing it here. I don't see any negative side effects of this remaining